pkg/metrics/status_test.go

```diff
@@ -131,34 +200,41 @@ func Test_statusCollector_Collect(t *testing.T) {
 		expectedCount        int
 	}{
 		{
-			name:                 "check status metrics",
+			name:                 "Success",
 			healthResponse:       sampleHealthResponse,
 			connectivityResponse: sampleSingleClusterConnectivityResponse,
+			expectedCount:        4,
+			expectedMetric:       expectedStatusMetricSuccess,
+		},
+		{
+			name:                 "Failure",
+			healthResponse:       sampleHealthResponse,
+			connectivityResponse: sampleSingleClusterUnreachableConnectivityResponse,
 			expectedCount:        5,
-			expectedMetric:       expectedStatusMetric,
+			expectedMetric:       expectedStatusMetricFailure,
 		},
 	}
 
 	for _, tt := range tests {
-		t.Log("Test :", tt.name)
-		collector := newStatusCollectorWithClients(&fakeDaemonClient{
-			response: tt.healthResponse,
-		}, &fakeConnectivityClient{
-			response: tt.connectivityResponse,
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Disable whitespace diff in github to reduce the size of the diff, the content here is unchanged, it only was wrapped in the `t.Run()`